stl/inc/algorithm

```diff
@@ -1263,6 +1263,43 @@ namespace ranges {
 } // namespace ranges
 #endif // __cpp_lib_concepts
 
+#ifdef __cpp_lib_concepts
+
+namespace ranges {
+    // ALIAS TEMPLATE copy_result
+    template <class _In, class _Out>
+    using move_result = in_out_result<_In, _Out>;
+
+    class _Move_fn : private _Not_quite_object {
+    public:
+        using _Not_quite_object::_Not_quite_object;
+
+        template <input_iterator _It, sentinel_for<_It> _Se, weakly_incrementable _Out>
+        requires indirectly_movable<_It, _Out> constexpr move_result<_It, _Out> operator()(

```

Comment:
Please disable clang-format for constrained functions as it cannot handle them currently.

requires is usually on a newline with 4 space indentation, constexpr at the following line

Reply:
I posted a short story with more detailed description for this in my review.